libraries/chain/include/steem/chain/account_object.hpp

```diff
@@ -49,6 +49,7 @@ namespace steem { namespace chain {
 
          bool              can_vote = true;
          util::manabar     voting_manabar;
+         uint16_t          voting_power = 0;
```

Comment:
You are adding this field to a consensus object that is never used, just to get an api call to build properly. The value is always 0, so replace `return vp_t1 ? vp_t1 : a.voting_power;` with `return vp_t1 ? vp_t1 : 0;`, which is equivalent to `return vp_t1;`.

Reply:
Hi @mvandeberg Thank you for your comment. To be honest, I don't know the very detail of the core blockchain. So I thought adding there and database_api_objects.hpp make it used by the database after examining other similar fields. But it isn't? Then, either please let me know some hints or fix it yourself if this can be done easily without a burden. still hope consistency. As far as I remember, before HF20, it was (not entirely sure though).

By the way, I'm fairly familiar with API (especially Python, I wrote a bot with it and it's working well) Now I'm also interested in the core, is there any good doc for it? Thanks.